src/threema.d.ts

```diff
@@ -495,6 +495,11 @@ declare namespace threema {
         isAll: boolean;
     }
 
+    interface WordResult {
+        word: string;
+        realLength: number;
```

Comment:
What exactly does `realLength` mean? From the tests it looks like it also includes trailing whitespace?

Reply:
Length of the untrimmed word

Reply:
I pushed a commit with some code comments: https://github.com/threema-ch/threema-web/pull/456/commits/22dde4b6c396acefd5c4f662aef2f8461184a321 Are they correct?

Reply:
:1st_place_medal: